doc/source/schemas/alleleAnnotations.rst

```diff
@@ -141,14 +141,14 @@ This protocol defines types used by the GA4GH Allele Annotation API.
   :field description:
     A description of the experiment.
   :type description: null|string
-  :field created:
+  :field createTime:
```

Comment:
Perhaps I'm being too pedantic, but "Time" to mean datetime or timestamp seems misleading. Note that there is an ISO 8601 Time that refers to just the time.

Could we make this createdDataTime or createdTimestamp? While we're at it, we should probably require UTC.